core/metacling/src/TClingBaseClassInfo.cxx

```diff
@@ -504,6 +504,11 @@ long TClingBaseClassInfo::Property() const
 
    clang::CXXBasePaths Paths(/*FindAmbiguities=*/false, /*RecordPaths=*/true,
                              /*DetectVirtual=*/true);
+
+   // This function might trigger deserialization due to auto library loading
```

Comment:
I think this first comment line is spurious - I do understand the second line and find it convincing :-) Please consider simply removing the first comment line.